### PR TITLE
Minor changes in Analytics Rule

### PR DESCRIPTION
**Pull request by vakohl**
Required items, please complete
   
   Change(s):
   - various minor changes in analytics rule

   Reason for Change(s):
   - rule validation

   Version Updated:
   - various versions

   Testing Completed:
   - syntactical testing done. Testing done for rules whose sample data was available.

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/Azure Web Application Firewall (WAF)/Analytic Rules/MaliciousWAFSessions.yaml [hunk 1]
```diff
@@ -38,7 +38,8 @@ query: |
       | where TimeGenerated > ago(queryperiod)
       | where Category == 'ApplicationGatewayAccessLog' and (isempty(httpStatus_d) or httpStatus_d in (successCode))
```
Comment (rahul0216) [inline Solutions/Azure Web Application Firewall (WAF)/Analytic Rules/MaliciousWAFSessions.yaml:39]: Do you think this comparison should also be case insensitive?

**Review by rahul0216 — COMMENTED** [review]
Please address the inline comment.

File: Solutions/Azure Web Application Firewall (WAF)/Analytic Rules/MaliciousWAFSessions.yaml (hunk 1 above)
Comment (vakohl) [reply Solutions/Azure Web Application Firewall (WAF)/Analytic Rules/MaliciousWAFSessions.yaml]: Thanks @rahul0216 . This has been updated to support case-insensitivity.

**Review by rahul0216 — APPROVED** [review]
Changes look good. Thanks

**Pull request merged**